### PR TITLE
[ES] Update The Team

### PR DESCRIPTION
**Pull request by Lalo611**
This time for sure, had a little mistake on the other pull request, so I closed it.
Reordered the sections just like in en.md.
Added to the core team:
-  -Mo-
-  Chaos
-  JBHyperion
-  juankristal
-  Mao
-  Nathanael
-  Okoratu
-  pishifat
-  Repflez
-  Toy
-  WalterToro

Moved to inactive core team:
-  cYsmix
-  deadbeat
-  HappyStick
-  Nyquill
-  RBRat3
-  Tom94
-  yelle

### CODE REVIEW

**Comment by Lalo611** [comment]
Forgot to write this:
Changed "Beatmap Nomination Team" to "Beatmap Nominators" from User groups.
Removed "Support Team Redux" from User groups.
Removed "Quality Assurance Team" from User groups.
Added "Nomination Assesmente Team" to User groups.
Added "Project Loved Team" to User groups.
Added "Support Team" to User groups.

File: wiki/People/The_Team/es.md [hunk 1]
```diff
@@ -1,75 +1,96 @@
...
+  - team
 ---
 
 # El Equipo
 
-Las personas que nombradas a continuación son el núcleo del equipo de osu!, y son los principales responsables de ayudar a que el juego sea impresionante para todos.
+*Para una lista de promociones y retiros, vea: [Staff Log](/wiki/Staff_Log)*
 
-## Creador
+Las personas nombradas a continuación son el núcleo del equipo de osu!, y son los principales responsables de ayudar a que el juego sea impresionante para todos.
```
Comment (cl8n) [inline wiki/People/The_Team/es.md:14]: should say something about how this is called "osu!team"

File: wiki/People/The_Team/es.md [hunk 2]
```diff
@@ -1,75 +1,96 @@
...
 
 | Nombre | Descripción |
```
Comment (cl8n) [inline wiki/People/The_Team/es.md:16]: second column doesn't match English (`Role`)

File: wiki/People/The_Team/es.md [hunk 3]
```diff
@@ -1,75 +1,96 @@
...
 | ![][flag_AU] [smoogipooo](https://osu.ppy.sh/users/1040328) | Desarrolador de osu!mania, cazador de bugs |
-| ![][flag_DE] [Tom94](https://osu.ppy.sh/users/1857058) | Desarrolador del pp |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Administrador de Project Loved, divulgador de artistas destacados, tutor de la comunidad |
 | ![][flag_ES] [Trosk-](https://osu.ppy.sh/users/3469385) | Soporte al jugador |
-| ![][flag_JP] [yelle](https://osu.ppy.sh/users/4916903) | Administrador de la [osu!store](https://osu.ppy.sh/store/listing) |
+| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | Ama de llaves, asistente de torneos, mantenimiento de la osu!wiki |
 | ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55) | Especie en peligro |
 
-## Inactivos
+## Grupos de Usuarios
```
Comment (cl8n) [inline wiki/People/The_Team/es.md:40]: sentence case for headers
```suggestion
## Grupos de usuarios
```

File: wiki/People/The_Team/es.md (hunk 1 above)
Comment (Lalo611) [reply wiki/People/The_Team/es.md:14]: It is already there, is just that I think that "equipo de osu!" fits better than "osu!equipo".
Let me now if you want me to change it to "osu!equipo".

**Review by TPGPL — APPROVED** [review]

**Pull request merged**